app/templates/content/cookies.html

```diff
@@ -106,6 +106,11 @@ <h3>Introductory message cookie</h3>
                 <td>Saves a message to let us know that you have seen our cookie message</td>
                 <td>1 month&nbsp;</td>
             </tr>
+            <tr>
+                <td>seen_user_research_message</td>
```

Comment:
Should this be put in it's section rather than a new row added to this section?

Reply:
Suggested content change (would need to be checked by the lawyer):

```Tells us that you have seen our cookie message```